metadata.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Metadata"
+description: "Configure and use metadata across the platform"
+---
+
+Metadata are a key data source for understanding and analyzing a workspace's customer feedback.
+
+There are 3 types of metadata we store and utilize in Monterey:
+
+- Feedback-Level Metadata: metadata that is tied to a specific feedback item, such as the version of the app of an iOS review
+- Customer-Level Metadata: metadata that tied to the customer providing feedback, such as a job title, username, or tracked user-event
+- Account-Level Metadata: metadata that is tied to the overarching account of a group of customers, such as an account contract value
```

Comment:
I'd suggest just "Feedback Metadata" (and Customer Metadata, Account Metadata) -- no `-level`.